youthful-piety/modernized/05-chapter-04.adoc

```diff
@@ -715,10 +718,15 @@ Many a snare does the enemy lay to entangle,
 but the good Spirit of God will discover all these.
 If you are enabled to do any good thing, do not depend upon that for acceptance.
 It is all through mercy, pure, unmerited mercy, that we are saved,
-by having our sins washed away in the blood of the Lamb.`"
+by having our sins washed away in the blood of the Lamb.
+____
 
+[.offset]
 To another, he said,
```

Comment:
I think the offset looks good here. You can leave it between the two paragraphs.